templates/core/index.html

```diff
@@ -1,1157 +1,744 @@
+      <hr class="p-rule" />
+      <h2>适用于智能设备与应用场景</h2>
+    </div>
+    <div class="grid-row--25-75 p-section--shallow">
+      <div class="grid-col">
+        <hr class="u-hide--small p-rule" />
+        <div class="grid-row">
+          <div class="grid-col-2 grid-col-medium-1">
+            <a href="https://canonical.com/solutions/automotive">
+              <h3 class="p-heading--5">汽车行业</h3>
+              <div class="p-image-container">
+                {{ image(url="https://assets.ubuntu.com/v1/f867c883-For-smart-use-cases__Automotivew.png",
+                                alt="",
+                                width="852",
+                                height="480",
+                                hi_def=True,
+                                loading="lazy",
+                                attrs={"class": "p-image-container__image"}) | safe
+                }}
+              </div>
+            </a>
+          </div>
+          <div class="grid-col-2 grid-col-medium-1">
+            <a href="https://canonical.com/solutions/industrial">
+              <h3 class="p-heading--5">工业</h3>
+              <div class="p-image-container">
+                {{ image(url="https://assets.ubuntu.com/v1/0162bc71-For-smart-use-cases__Industrial.png",
+                                alt="",
+                                width="852",
+                                height="480",
+                                hi_def=True,
+                                loading="lazy",
+                                attrs={"class": "p-image-container__image"}) | safe
+                }}
+              </div>
+            </a>
+          </div>
+          <div class="grid-col-2 grid-col-medium-1">
+            <a href="https://ubuntu.com/robotics">
+              <h3 class="p-heading--5">机器人</h3>
+              <div class="p-image-container">
+                {{ image(url="https://assets.ubuntu.com/v1/aeeeb32d-For-smart-use-cases__Robotics.png",
+                                alt="",
+                                width="852",
+                                height="480",
+                                hi_def=True,
+                                loading="lazy",
+                                attrs={"class": "p-image-container__image"}) | safe
+                }}
+              </div>
+            </a>
+          </div>
+        </div>
       </div>
     </div>
-  </section>
-
-  <section class="p-strip--image p-banner-core" id="reduce-the-cost">
-    <div class="row u-vertically-center">
-      <div class="col-8">
-        <h2>降低Linux成本</h2>
-        <p class="p-heading--4">Ubuntu Core免费使用，额外服务需付费。</p>
-        <p>Ubuntu Core参考镜像可以免费重新分发。与Canonical合作获取硬件适配和定制的服务级别。我们会让您更快地进入市场。</p>
+    <div class="grid-row--25-75">
+      <div class="grid-col">
+        <hr class="u-hide--small p-rule" />
+        <div class="grid-row">
+          <div class="grid-col-2 grid-col-medium-1">
+            <a href="/internet-of-things/smart-displays">
```

Comment:
Are you sure it should point to the current website's page, as in the copy doc it's pointing at ubuntu.com's one? If this is confirmed by the stakeholder, the change should also be done in the copy doc.